Source/DafnyCore/TestGenerationOptions.cs

```diff
@@ -10,8 +10,13 @@ public class TestGenerationOptions {
     public enum Modes { None, Block, Path };
     public Modes Mode = Modes.None;
     [CanBeNull] public string TargetMethod = null;
-    public uint? SeqLengthLimit = null;
+    public int SeqLengthLimit = -1;
```

Comment:
This is a step backwards in terms of clarity IMO, what's the motivation?

Reply:
I changed `null` to `-1` here so that is would be consistent with the [new CLI](https://github.com/dafny-lang/dafny/blob/72bf43a506aea1a0e7cacf63c18b53f41167bbb7/Source/DafnyTestGeneration/GenerateTestsCommand.cs#L62-L74), which uses `Option<int>`. Not sure what's the best way around this is.

Reply:
I have updated this option to be of type `uint` and to use `0` for default as per @atomb's suggestion - I think this should be better on all counts.